src/lib/features/project/project-store.e2e.test.ts

```diff
@@ -167,3 +167,62 @@ test('should add environment to project', async () => {
 
     expect(envs).toHaveLength(1);
 });
+
+test('should update project enterprise settings', async () => {
```

Comment:
didn't we have existing tests in this repo or the enterprise repo?

Reply:
No. We only have scheme test in Enterprise